ext/featurefmt/apk/apk.go

```diff
@@ -74,11 +76,20 @@ func (l lister) ListFeatures(files analyzer.Files) ([]database.FeatureVersion, e
 				if len(libToDeps) != 0 {
 					pkg.LibraryToDependencies = libToDeps
 				}
+				if source != "" {
+					pkg.Feature.Name = source
+					source = "" // This can be moved outside the outer `if` where the other variables are reset
```

Comment:
nit: that comment was more of a note for you. Let's either remove the comment or move `source = ""` down